src/game.cpp

```diff
@@ -115,6 +115,38 @@ const skill_id skill_firstaid( "firstaid" );
 const species_id ZOMBIE( "ZOMBIE" );
 const species_id PLANT( "PLANT" );
 
+const efftype_id effect_adrenaline_mycus( "adrenaline_mycus" );
```

Comment:
Why are we adding MORE stuff to game.cpp? This file is already too big and we shouldn't be cramming stuff into it.